dev-docs/source/Testing/SliceViewer/SliceViewer.rst

```diff
@@ -181,9 +184,9 @@ Test the :ref:`Peak Overlay<mantid:sliceviewer_peaks_overlay>`
 3. Double click a row
 
     - It should change the slicepoint along the integrated momentum axis and zoom into the peak - e.g. in (X,Y) = (H,K) then the slicepoint along L will be set to 1 and there will be a cross at (0,0).
-    - Note for ``md_4D`` the cross should be plotted at all E (obviously a Bragg peak will only be on the elastic line but the peak object has no elastic/inelastic logic and the sliceviewer only knows that `E` is not a momentum axis, it could be temperature etc.).
+    - Note for ``md_4D`` a peak will not be plotted if a non-Q axis is viewed, if both axes are Q-dimensions then the cross should be plotted at all E (obviously a Bragg peak will only be on the elastic line but the peak object has no elastic/inelastic logic and the sliceviewer only knows that `E` is not a momentum axis, it could be temperature etc.).
```

Comment:
I must confess I got confused here, because initially I thought that I was not supposed to see the red cross at all for `md_4D` workspace. It was after playing around with a combination of different axes that I realised what you meant. This makes perfect sense as it is, so I don't think it is worth changing it.